addons/mail/static/src/web/chat_window/chat_window.js

```diff
@@ -110,6 +125,11 @@ export class ChatWindow extends Component {
         this.chatWindowService.notifyState(this.props.chatWindow);
     }
 
+    togglePinMenu() {
+        this.state.activeMode =
+            this.state.activeMode === "pinned-messages" ? "" : "pinned-messages";
+    }
+
```

Comment:
Out of scope of PR, but we clearly see some code duplication in showing of menu in discuss topbar / chatwindow header. Could be worth doing some code cleaning after this PR at one point.